exe/wallet/http-bridge/Main.hs

```diff
@@ -147,6 +155,7 @@ main = do
 -- | Arguments for the 'launch' command
 data LaunchArgs = LaunchArgs
     { _network :: Either Local Network
+    , _addressScheme :: AddressScheme
```

Comment:
This doesn't sound right and is out-of-scope for now. We'll have to support random and sequential wallets at the same time. There's no choosing between both, as clients want the ability to use both on a single wallet server.

Reply:
I know it's not ideal but I thought adding a command-line switch would be a good intermediate step. See the comments in the PR description.